### DISCUSSION TITLE
Slash teams in multiple guilds

### DISCUSSION THREAD

**Discussion by IIDarkNessYT** [Rapptz/discord.py#8220] [Q&A] [answered]
Hello! I would like to ask how do I use slash commands for multiple guilds? To bot works in all invited guilds

**Comment by GoogolGenius** [comment, marked as answer]
Your question is a little confusing, however, I'll try my best to answer:

If you want to make an app_command for a specific guild(s) you can use the [`app_commands.guilds`](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.guilds) decorator, then call [`CommandTree.sync()`](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.CommandTree.sync) with the guild(s) you would like to sync those commands to.

If you're talking about using app_commands in all guilds meaning globally, then you omit the [`app_commands.guilds`](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.guilds) decorator and call [`CommandTree.sync()`](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.CommandTree.sync) without any parameters.

**Reply by IIDarkNessYT** [reply]
Thank's!
